tools/mapping-tester/compare.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+
+import argparse
+from sys import exit
+
+import polars as pl
```

Comment:
There seems to be a problem with polars being called via `ctest`: in my venv it works locally (using `run.sh`), but `ctest` fails for me as well.

Reply:
A problem is that we currently overwrite the PATH at configuration time. So any further modifications are not picked up.

I just saw that CMake 3.22 (our new baseline) provide ENV modification support for exact this use-case.

Reply:
I am wondering why it worked or works for the precice-profiling script (also used in the mapping tester), whereas it fails for the compare script. The precice-profiliing script is called through the gathering script `os.subprocess`, maybe that's the simple answer.

Reply:
In my venv, the tests pass now. The CI doesn't seem to be happy

https://github.com/precice/aste/actions/runs/11552451140/job/32151545870?pr=212#step:10:447

I am not sure if the actual solution here would be more simple.

Reply:
> Isn't this a property from the mapping accuracy that should be unchanged?

I agree. Maybe the VTK precision or version is different?

Reply:
ASTE generates, exports, partitions to VTU with binary data and uses the VTK-internal calculator.

The only moving targets are the VTK version and the numpy version, which we use to calculate the [percentiles](https://numpy.org/doc/stable/reference/generated/numpy.percentile.html)/median.

Maybe they changed the interpolation method?

Reply:
The [assert checks col by col](https://github.com/pola-rs/polars/blob/d51b12c706983b423e8ad344a5636ae903fe3431/py-polars/polars/testing/asserts/frame.py#L105) and the first column is the median. So it is possible that other columns don't match too.

Reply:
Do you have the complete result table for both runs?

Reply:
Could it be a matter of the column order or something?